inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+little lock on the annotation page) are taken into account.
+** *Evaluation*: The evaluation page shows a learning courve diagram of
+each recommender (we will explain recommenders later on on this guide).
+** *Settings*: This is the heart of your project. Here, you can
+organize, manage and adjust all details of your project. We will look at
+the ones you need to get started for your own projects. For a quick
+overview on the settings, you might want to watch our tutorial video
+„Overview“:
+https://www.youtube.com/watch?v=wp4AN3p23mQ&list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&index=3&t=0s.
 
-Running {product-name} in the way you just did is a great way to get started and try out its
-capabilities, but it is not the best way of working with the application. If you like 
-{product-name},  please be sure to check out the <<admin-guide.adoc,Administrator Guide>> to 
-learn how to set up a production-ready instance.
+* Each project consist of
+** One or more *documents* to annotate
+** *Users* to work on the project
+** *Layers* to annotate with
+** Optional: one or more *Knowledge Base/s*
+** Optional: *Recomenders* to automatically suggest annotations
+** Optional: *Guidelines* for your team
 
-[NOTE]
-====
-By default, {product-name} creates and uses an embedded database and stores all its data in  folder called `.inception` (_dot inception_) within your home folder. While this allows you to get started very quickly in trying out the application, it is not a recommended configuration for serious use. For production use, please configure {product-name} to use a database server when using it in production. For more information, please refer to the <<admin-guide.adoc#sect_database,Administrator Guide>>.
-====
+and you find all of these in the project *settings* just mentioned.
+Again, for details on any section/topic mentioned, please check the main
+documentation.
 
-[NOTE]
-====
-By default the server starts on port 8080 and you can access it via a browser at `http://localhost:8080` after you started it. But if you already have a service running on that port, you can add the parameter `-Dserver.port=9999` at the end of the command line to start the server on port 9999 (or choose any other port).
-====
+* In the section **Documents, **you may upload your files to be
+annotated. Make sure that the format selected (see the dropdown on the
+right) is the same as the file.
+* In the section **Users, **you may add users to your project and change
+their rights in this project. You can only add existing users here from
+the dropdown (left).
+** In order to *add new users* so they can be chosen from this dropdown,
+click on „*administration*“- button in the very top right corner and
+select section *Users* on the left. For *user roles* (within the whole
+instance of INCEpTION) see
+https://zoidberg.ukp.informatik.tu-darmstadt.de/jenkins/job/INCEpTION%20(GitHub)%20(master)/de.tudarmstadt.ukp.inception.app$inception-app-webapp/doclinks/1/#sect_users
```

Comment:
Make it an asciidoc link an use the recent link
https://inception-project.github.io/documentation/latest/developer-guide